pyproject.toml

```diff
@@ -3,3 +3,11 @@ line-length = 120
 
 [tool.isort]
 profile = "black"
+line_length = 120
```

Comment:
# Reviewer Note

Consolidating this from its previously separate file into this one